pylustrator/components/matplotlibwidget.py

```diff
@@ -77,7 +82,8 @@ def schedule_draw(self):
     def draw(self):
         self.timer.stop()
         import traceback
```

Comment:
I guess instead of adding an empty line after the import this import should go to the top.